### ISSUE TITLE
Does this figure mean that the fuzzer tool is running normally？？

### ISSUE THREAD

**Issue by Mrxia111** [alipay/Owfuzz#10]
![测试图](https://user-images.githubusercontent.com/92384670/156982764-fa53b213-7461-43a6-90b0-44dedb1f40ad.png)
This interface keeps flashing. I don't know how to operate it. Can you give me some suggestions on how to use this tool next.

**Comment by E7mer** [comment]
Yes, it's normal. Next, you need to pay attention to the ”PoC Count“， and try to reproduce them to see if they're real.
